common/src/traits.rs

```diff
@@ -0,0 +1,80 @@
+//! Commonly used traits
```

Comment:
maybe we name this module "filters"?

Reply:
I think we'll eventually move more traits in here so can stick with traits and then make a filter submodule when we add more